reviewdog.yml

```diff
@@ -0,0 +1,3 @@
+runner:
+  golint:
+    cmd: golint ./...
```

Comment:
Does this command also need to filter out `vendor`? I'm not really sure what this command is used for TBH.